isis/src/docsys/documents/InstallGuide/InstallGuide.xml

```diff
@@ -24,30 +24,124 @@ you must <em>not</em> upgrade the ISIS Data Files!!!
 -->
   <h2>Overview</h2>
 
+  <p>
+    This installation guide is for ISIS3 users interested in installing ISIS3 through conda, or compiling on their own systems using cmake.  
```

Comment:
Should we mention the "compiling on their own systems using cmake"? That seems more developer-oriented.

It also may be useful to say that this installation guide is for ISIS3 users interesting in installing ISIS3 (3.6.0+) through conda.

Reply:
I think we should delete that part.